src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -335,21 +449,18 @@ private String consolidateDataSetElectionsResult(List<Election> datasetElections
     }
 
     private List<Document> getUnReviewedDarsForUser(AuthUser authUser) {
+        List<Document> activeDars = getAllDataAccessRequestsAsDocuments().stream().
+                filter(d -> !ElectionStatus.CANCELED.getValue().equalsIgnoreCase(d.getString(DarConstants.STATUS))).
+                collect(Collectors.toList());
```

Comment:
Note: Add a story to allow for querying into the json in postgres so we don't have to stream over all DARs.

Reply:
Added https://broadinstitute.atlassian.net/browse/DUOS-609